cms/models.py

```diff
@@ -210,7 +240,7 @@ class FrequentlyAskedQuestion(Orderable):
     """
     FAQs for the program
     """
-    program_page = ParentalKey(ProgramPage, related_name='faqs')
+    faqs_page = ParentalKey(FaqsPage, related_name='faqs', null=True)
```

Comment:
I think the migration reverse problem comes from this change. `program_page` doesn't allow null values so when it's added back with a null default, it causes problems. I don't think there's anything we can do about this, we just have to live with the migration being non-reversable. Any thoughts @giocalitri?